posthog/queries/test/test_trends.py

```diff
@@ -4929,7 +4929,8 @@ def test_breakdown_multiple_cohorts(self):
             groups=[{"properties": [{"key": "key_2", "value": "value_2", "type": "person"}]}],
         )
 
-        cohort1.calculate_people_ch(pending_version=0)
+        # try different versions
+        cohort1.calculate_people_ch(pending_version=1)
```

Comment:
confused how this test passed before then 🤔 , since if the params were the same, the result should've been the same?

Reply:
it did pass before because both arguments were 0. I update the params to be different so it shouldn't pass without this fix in this PR